src/app/verification/page.tsx

```diff
@@ -0,0 +1,26 @@
+import ControlOtp from '@/components/pages/verification/control-otp'
+import Content from '@/components/ui/content'
+import { Icons } from '@/components/ui/icons'
+import ImageCard from '@/components/ui/image-card'
+import { ThemeToggler } from '@/components/ui/theme-toggler'
+
+export default function Verification() {
+  return (
+    <section className="h-screen w-screen bg-gradient-blue-white">
+      <Content className="flex items-center justify-center dark:bg-none dark:bg-base-gray-8">
+        <div className="p-7 sm:p-10 flex gap-7 md:gap-[3.75rem] rounded-3xl bg-base-white dark:bg-base-black">
+          <ImageCard className="-my-[4.375rem] flex-shrink-0 hidden sm:flex pt-24 pr-8 pb-36 pl-5">
```

Comment:
As I see, classes '-my-[4.375rem] flex-shrink-0 hidden sm:flex' repeated in each image-card render. Should we place them to component's className instead?

Reply:
`flex-shrink-0` can be placed in the component itself, but other classes are specific due to the component place. For example margin or display styles like those should be only on the auth pages. Or padding, which could be different

Reply:
So as I think, we should create variants like in a button.tsx 
variants: {
    auth: '......',
    default: '.......',
},

Reply:
Not a bad idea, I'll try it out and see if it is not cringey 😄

Reply:
Haha, okay